org-cyf-itp/content/onboarding/success/index.md

```diff
@@ -0,0 +1,32 @@
++++
+title = 'success'
+description = 'How do we know we have completed User Focused Data?'
+layout = 'success'
+emoji= '✅'
+menu_level = ['module']
+weight = 11
+[[objectives]]
+1="Link to your PR for [Wireframe to Web Code](https://github.com/CodeYourFuture/Module-User-Focused-Data/issues/90)"
+2="Link to your PR for [Form Controls](https://github.com/CodeYourFuture/Module-User-Focused-Data/issues/88)"
+3="Link to your Google Sheet for [Query Data in Google Sheets with SQL](https://github.com/CodeYourFuture/Module-User-Focused-Data/issues/104)"
++++
+
+### 🎯 You've achieved your learning objectives if you can:
+
+```objectives
+- Track changes to data (your code) over time with Git and GitHub.
+- Structure data for presentation and user input with HTML and HTML forms.
+- Query and analyse structured data with SQL in a spreadsheet.
+```
+
+### 💯 To complete _any_ module, you must:
+
+```objectives
+- Do your work: complete all mandatory backlog issues
+- Come to class
+- **Submit** the Dashboard step for this module
+```
+
+## ✅ To complete _this_ module, you must:
```

Comment:
I haven't changed this page yet because I want the step by step guide I've mentioned. 

I find the dashboard incomprehensible so I'm struggling to get these views together.